### PR TITLE
Parkwarden loop complete! + extra

### PR DESCRIPTION
**Pull request by Ryan-Myers**

### CODE REVIEW

**Review by DavidSM64 — APPROVED** [review]
Don't see any notable issues. Feel free to merge when you're ready.

File: src/object_functions.c [hunk 1]
```diff
@@ -1967,7 +2567,7 @@ void obj_loop_goldenballoon(Object *obj, s32 speed) {
                     obj64->unkC = obj64->unkD;
                 }
             } else {
-                func_8001C6C4(obj64, obj, sp2C, speedf, 0);
+                func_8001C6C4((Object_64 *)obj64, obj, sp2C, speedf, 0);
```
Comment (FazanaJ) [inline src/object_functions.c:2570]:
```suggestion
                func_8001C6C4((Object_64 *) obj64, obj, sp2C, speedf, 0);
```

File: src/object_functions.c [hunk 2]
```diff
@@ -2876,7 +3476,7 @@ void obj_loop_bubbler(Object *obj, s32 speed) {
 }
 
 void obj_init_boost(Object *obj, LevelObjectEntry_Boost *entry) {
-    obj->unk64 = (s32) get_misc_asset(MISC_ASSET_UNK14) + (entry->unk8[0] << 7);
+    obj->unk64 = (Object_64 *)((s32) get_misc_asset(MISC_ASSET_UNK14) + (entry->unk8[0] << 7));
```
Comment (FazanaJ) [inline src/object_functions.c:3479]:
```suggestion
    obj->unk64 = (Object_64 *) ((s32) get_misc_asset(MISC_ASSET_UNK14) + (entry->unk8[0] << 7));
```

File: src/objects.c [hunk 3]
```diff
@@ -799,7 +799,7 @@ s32 func_8000FBCC(Object *arg0, Object_60 *arg1) {
     arg1->unk4 = NULL;
     objHeader = ((ObjectSegment*) arg0)->header;
     if (objHeader->unk32) {
-        arg1->unk4 = load_texture((s32) ((Object_Taj*)objHeader)->unk34);
+        arg1->unk4 = load_texture((s32) ((ObjectHeader*)objHeader)->unk34);
```
Comment (FazanaJ) [inline src/objects.c:802]:
```suggestion
        arg1->unk4 = load_texture((s32) ((ObjectHeader *) objHeader)->unk34);
```

**Pull request merged**